src/it/resources/com/google/checkstyle/test/chapter4formatting/rule412nonemptyblocks/InputRightCurlySwitchCasesBlocks.java

```diff
@@ -72,15 +78,71 @@ public static void test3() {
     int k = 0;
     switch (k) {
       case 1:
-        { // violation ''{' at column 9 should be on the previous line.'
+        {
           int x = 1;
           break;
         }
-      case 2:
-        { // violation ''{' at column 9 should be on the previous line.'
-        int x = 2; } // violation ''}' at column 20 should be alone on a line.'
+      default:
+        {
+          int x = 2; } // false-negative
```

Comment:
Why is it false negative?

Reply:
Sorry at the time I sent this PR I was also working on KR style block issue, so I mistakenly added `false-negative` comment as we weren't able to detect KR block, but let's keep this comment as we're not able to detect KR block and I will be working on that issue in near future. It'll help me to fix the edge cases for easily

Reply:
Could you give a link to that issue?

Reply:
https://github.com/checkstyle/checkstyle/issues/15664